rdkit/Chem/RegistrationHash.py

```diff
@@ -127,13 +126,20 @@ def GetMolLayers(original_molecule: Chem.rdchem.Mol, data_field_names: Optional[
   mol = _RemoveUnnecessaryHs(original_molecule, preserve_stereogenic_hs=True)
   _StripAtomMapLabels(mol)
 
+  cxflag = Chem.CXSmilesFields.CX_ATOM_LABELS | Chem.CXSmilesFields.CX_ENHANCEDSTEREO
```

Comment:
I wonder if we should have some function to "clean" the atom labels by only keeping those we care about (e.g. Pol and Mol) and discarding the rest.

Reply:
`*`, or anything non-standard provided by the user, e.g.: `CCCCC-* |$"made_up";;;;;star_e$|`

![image](https://user-images.githubusercontent.com/7498185/223845485-4790c331-cb65-4c1c-a608-3dad59dd67f1.png)

Reply:
What is the difference between "made_up" and "Pol" or "Mod"

Reply:
"Pol" and "Mod" come from the Mol file standard, "made_up" is, well, made up. It's the first thing that came to my mind as some random thing I could use as a label.

In any case, answering to my own question, I think it should be the user's responsibility to clean up their mols before asking for the registration hash.

Reply:
Yeah, I think that answer is what I was trying to get at too. I think the user should pick/control that